src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -40,7 +40,7 @@ public JsonAdaptedPerson(@JsonProperty("name") String name, @JsonProperty("phone
         this.name = name;
         this.phone = phone;
         this.email = email;
-        this.address = address;
+        this.mod = address;
```

Comment:
Should this be address? I can't see the constructor arguments so I'm not sure of the constraints behind this as well.

Reply:
will fix!